charts/airbyte-cron/templates/deployment.yaml

```diff
@@ -0,0 +1,185 @@
+        {{- end }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" (dict "value" .Values.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.affinity }}
+      affinity: {{- include "common.tplvalues.render" (dict "value" .Values.affinity "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.extraInitContainers }}
+      initContainers:
+      {{- toYaml .Values.extraInitContainers | nindent 6 }}
+      {{- end }}
+      containers:
+        - name: airbyte-cron
+          image: {{ printf "%s:%s" .Values.image.repository (include "cron.imageTag" .) }}
+          imagePullPolicy: "{{ .Values.image.pullPolicy }}"
+          env:
+          {{- if eq .Values.global.deploymentMode "oss"  }}
+          - name: AIRBYTE_ROLE
+            valueFrom:
+              configMapKeyRef:
+                name: {{ .Release.Name }}-airbyte-env
+                key: AIRBYTE_ROLE
+          - name: AIRBYTE_VERSION
+            valueFrom:
+              configMapKeyRef:
+                name: {{ .Release.Name }}-airbyte-env
+                key: AIRBYTE_VERSION
+          - name: CONFIGS_DATABASE_INITIALIZATION_TIMEOUT_MS
+            valueFrom:
+              configMapKeyRef:
+                name: {{ .Release.Name }}-airbyte-env
+                key: CONFIGS_DATABASE_INITIALIZATION_TIMEOUT_MS
+          - name: CONFIGS_DATABASE_MINIMUM_FLYWAY_MIGRATION_VERSION
+            valueFrom:
+              configMapKeyRef:
+                name: {{ .Release.Name }}-airbyte-env
+                key: CONFIGS_DATABASE_MINIMUM_FLYWAY_MIGRATION_VERSION
+          - name: DATABASE_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.global.database.secretName | default (printf "%s-airbyte-secrets" .Release.Name ) }}
+                key: {{ .Values.global.database.secretValue | default "DATABASE_PASSWORD" }}
+          - name: DATABASE_URL
+            valueFrom:
+              configMapKeyRef:
+                name: {{ .Release.Name }}-airbyte-env
+                key: DATABASE_URL
+          - name: DATABASE_USER
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Release.Name }}-airbyte-secrets
+                key: DATABASE_USER
+          - name: MICRONAUT_ENVIRONMENTS      
+            valueFrom:
+              configMapKeyRef:
+                name: {{ .Release.Name }}-airbyte-env
+                key: CRON_MICRONAUT_ENVIRONMENTS
+          - name: REMOTE_CONNECTOR_CATALOG_URL
+            valueFrom:
+              configMapKeyRef:
+                name: {{ .Release.Name }}-airbyte-env
+                key: REMOTE_CONNECTOR_CATALOG_URL
+          - name: TEMPORAL_CLOUD_CLIENT_CERT
```

Comment:
Since this is the helm OSS chart we shouldn't support using temporal cloud here since it is not added to the worker. So all the env like `TEMPORAL_CLOUD_*` should be removed.